platform/mellanox/mlnx-platform-api/sonic_platform/chassis.py

```diff
@@ -27,7 +27,6 @@
 
 MLNX_NUM_PSU = 2
 
-GET_HWSKU_CMD = "sonic-cfggen -d -v DEVICE_METADATA.localhost.hwsku"
 GET_PLATFORM_CMD = "sonic-cfggen -d -v DEVICE_METADATA.localhost.platform"
```

Comment:
You can remove this line too, right? Since this command is not used any more either.

Reply:
fixed.